### ISSUE TITLE
Vulnerabilities report

### ISSUE THREAD

**Issue by daudmalik06** [knex/knex#5558]
# Environment

Knex version: main branch
OS: centos

# Bug

Hello maintainers,

I wanted to bring to your attention that after installing the package, I ran a vulnerability scan with vulert abom on the package-lock.json file and discovered that there are 3 vulnerable dependencies present. As these vulnerabilities can potentially impact the security of the entire project, I am unsure whether to report this under responsible disclosure.

For your reference, here is a link to the report of the package-lock file I scanned: https://vulert.com/vuln-scan/list/737ae7a2-eeca-4541-acaa-7f0f3ef68e9f

I recommend that we take immediate action to address these vulnerabilities and ensure the security of the project. Please let me know if you require any further information.

Thank you.

**Comment by mercmobily** [comment]
We will need to check that the audit is clean when making the first release. Marking this as IMPORTANT so that it's done ASAP

**Comment by daudmalik06** [comment]
> We will need to check that the audit is clean when making the first release. Marking this as IMPORTANT so that it's done ASAP

@mercmobily Please do not hesitate to contact us for assistance with your audit requirements. We have extensive experience in conducting **audits, penetration testing, and vulnerability assessments**. Our email address is Info@devnack.com.

**Comment by myndzi** [comment]
These are all test dependencies, so they aren't impactful to users of the library thankfully.

**Comment by erulabs** [comment]
This appears fixed by https://github.com/knex/knex/commit/c92bbb3d30676b564bb5b6b09488dda1d6b11475

**Comment by myndzi** [comment]
Makes sense, that change was made to clean up a number of `npm audit` warnings, but I didn't actually go check if any specific issues were linked to it. Thanks for the heads up :)
